test/parallel/test-fs-promises-read-file.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const common = require('../common');
+
+const assert = require('assert');
+const path = require('path');
+const { writeFile, readFile } = require('fs/promises');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const fn = path.join(tmpdir.path, 'large-file');
+
+common.crashOnUnhandledRejection();
+
+// Creating large buffer with random content
+const buffer = Buffer.from(
```

Comment:
You can generate this buffer with `crypto.randomBytes(16834 * 2)` actually, but the current method LGTM anyway.